backend/src/handlers/BaseHandler.ts

```diff
@@ -25,43 +24,53 @@ export abstract class BaseHandler<TPayload, TOutput extends BaseOutput> implemen
   /**
    *  @param validationSchema - The Joi schema that the payload will be validated against
    */
-  constructor(private validationSchema: Joi.ObjectSchema<TPayload>, private enumType: { [key: string]: string }) {}
+  private validationSchema: Joi.ObjectSchema<TPayload>;
+  private clientErrorCodes: { [key: string]: string };
+  protected requiresAuthentication: boolean;
+
+  constructor(
+    validationSchema: Joi.ObjectSchema<TPayload>,
+    clientErrorCodes: { [key: string]: string },
+    requiresAuthentication: boolean = true
+  ) {
+    this.validationSchema = validationSchema;
+    this.clientErrorCodes = clientErrorCodes;
+    this.requiresAuthentication = requiresAuthentication;
```

Comment:
@tommy-trombone and @j-phey we are just passing a flag as `false` from our handlers that don't require auth and things will work (/ skip auth checks). 

You will also get the `req` as third optional param in your `getResult` function if you aren't using auth.